lib/src/static_file.dart

```diff
@@ -27,11 +27,11 @@ class StaticFile extends Interceptor {
   const StaticFile({this.charset: 'utf-8', this.encoding: UTF8});
 
   @InputRouteResponse()
-  Future<Response<Stream>> post(Response<JaguarFile> response) async {
+  Future<Response<dynamic>> post(Response<JaguarFile> response) async {
```

Comment:
Should be changed back to Response<Stream>. For accommodating file not found, look at comment below.